config.json

```diff
@@ -84,6 +84,18 @@
           "word-definition"
         ]
       },
+      {
+        "slug": "acronym",
+        "name": "Acronym",
+        "uuid": "e8733850-cc86-4d29-8e5a-92078576e193",
+        "practices": [
+          "strings",
+          "quotations",
+          "collections"
+        ],
+        "prerequisites": [],
```

Comment:
Are there no prerequisites? It looks like the student _has_ to know about strings, so maybe add that as a prerequisite?

Reply:
I'll come back after this and add prerequisites - I think in Factor there's nothing yet